modules/hello-world-impl/src/main/java/org/opencastproject/helloworld/impl/HelloWorldServiceImpl.java

```diff
@@ -42,9 +47,28 @@
   /** The module specific logger */
   private static final Logger logger = LoggerFactory.getLogger(HelloWorldServiceImpl.class);
 
+  /** Message to print */
+  private String message = "Hello World";
```

Comment:
That default here is unnecessary, right? Since we already use one in activate(). Or does the compiler complain if we leave this out?

```suggestion
  private String message;
```

Reply:
The tests fail without this, if nothing else.